tools/pyi/gen_pyi.py

```diff
@@ -171,6 +171,12 @@ def should_bind_method(python_func: PythonSignatureNativeFunctionPair) -> bool:
     "copy_",
 ]
 
+overload_blocklist = [
+    # Overloads which are more specific than another, so not needed
+    "all.dim",
```

Comment:
nit. Put the name of the other overload.

Reply:
wdym? If we skip the array overload then mypy would complain when you pass a list of dims.